### DISCUSSION TITLE
Trigger query [Jun 02]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2310] [Discord Threads]
> 🙋‍♂️ **duongtu**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### We support webhook. You can find details here   
#### https://docs.nocodb.com/developer-resources/webhooks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/981457726355931166)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
